src/apps/backend/modules/booking/types.ts

```diff
@@ -0,0 +1,38 @@
+import { ToolCall } from '../common-types/common';
+
+export class PhoneNumber {
+  countryCode: string;
+  phoneNumber: string;
+
+  constructor(countryCode: string, phoneNumber: string) {
+    this.countryCode = countryCode;
+    this.phoneNumber = phoneNumber;
+  }
+}
+
+export interface CreateBookingRequestParams {
+  address: Address;
+  firstName: string;
+  lastName?: string;
+  phoneNumber: PhoneNumber;
+  schedule: ScheduleTime;
+}
+
+export interface BookingCustomRequestBody {
```

Comment:
This type should be defined in vapi module.

Vapi controller should deconstruct the vapi specific request typedef, and convert into appropriate type that booking service method expects.